--- /usr/local/lib/python3.6/site-packages/graphsurgeon/node_manipulation.py	2022-09-13 11:20:04.792472821 +0200
+++ /usr/local/lib/python3.6/site-packages/graphsurgeon/node_manipulation.py	2022-09-13 11:20:50.796467758 +0200
@@ -86,6 +86,7 @@
     '''
     node.name = name or node.name
     node.op = op or node.op or node.name
+    node.attr["dtype"].type = 1
     for key, val in kwargs.items():
         if isinstance(val, tf.DType):
             node.attr[key].type = val.as_datatype_enum
